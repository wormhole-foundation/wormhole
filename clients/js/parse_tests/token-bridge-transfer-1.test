010000000001007d204ad9447c4dfd6be62406e7f5a05eec96300da4048e70ff530cfb52aec44807e98194990710ff166eb1b2eac942d38bc1cd6018f93662a6578d985e87c8d0016221346b0000b8bd0001c69a1b1a65dd336bf1df6a77afb501fc25db7fc0938cb08595a9ef473265cb4f0000000000000003200100000000000000000000000000000000000000000000000000000002540be400165809739240a0ac03b98440fe8985548e3aa683cd0d4d9df5b5659669faa3010001000000000000000000000000c10820983f33456ce7beb3a046f5a83fa34f027d00020000000000000000000000000000000000000000000000000000000000000000