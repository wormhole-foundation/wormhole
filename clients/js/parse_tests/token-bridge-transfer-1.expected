{
  version: 1,
  guardianSetIndex: 0,
  signatures: [
    {
      guardianSetIndex: 0,
      signature: '7d204ad9447c4dfd6be62406e7f5a05eec96300da4048e70ff530cfb52aec44807e98194990710ff166eb1b2eac942d38bc1cd6018f93662a6578d985e87c8d001'
    }
  ],
  timestamp: 1646343275,
  nonce: 47293,
  emitterChain: 1,
  emitterAddress: '0xc69a1b1a65dd336bf1df6a77afb501fc25db7fc0938cb08595a9ef473265cb4f',
  sequence: 3n,
  consistencyLevel: 32,
  payload: {
    module: 'TokenBridge',
    type: 'Transfer',
    amount: 10000000000n,
    tokenAddress: '0x165809739240a0ac03b98440fe8985548e3aa683cd0d4d9df5b5659669faa301',
    tokenChain: 1,
    toAddress: '0x000000000000000000000000c10820983f33456ce7beb3a046f5a83fa34f027d',
    chain: 2,
    fee: 0n
  },
  digest: '0x2862e5873955ea104bb3e122831bdc43bbcb413da5b1123514640b950d038967'
}
